torchgen/model.py

```diff
@@ -2603,9 +2607,9 @@ def __post_init__(self) -> None:
                 " See Note [view_copy NativeFunctions] for details."
             )
         else:
-            assert self.view_copy.func.name.name.base.endswith("_copy")
```

Comment:
Nit: keep this assert, and assert that it ends with either `_copy` or `_scatter`?

Reply:
sure that works!